css/src/core/focus.scss

```diff
@@ -0,0 +1,30 @@
+@if ($focus-visible-use-polyfill) {
+	.js-focus-visible .has-default-focus *:focus:not(.focus-visible) {
+		outline: none;
+	}
+
+	.focus-visible.has-inner-focus {
+		outline-offset: -$focus-outline-offset;
+	}
+
+	.has-default-focus {
+		.is-focused,
+		.focus-visible {
+			@include focus();
+
+			&.has-inner-focus {
+				outline-color: currentColor;
+				outline-offset: -$focus-outline-offset;
+			}
+		}
+	}
+} @else {
+	*:focus-visible {
+		@include focus();
```

Comment:
@extend %focus;

Reply:
Updated, not sure why the font's showing as black on Github though.